Java/core/src/main/java/com/amazon/randomcutforest/tree/AbstractCompactRandomCutTree.java

```diff
@@ -575,4 +575,8 @@ public T maxSize(int maxSize) {
 
     }
 
+    @Override
+    protected void changeCacheFraction(double fraction) {
+        boxCache.reInitialize(fraction);
```

Comment:
minor: I prefer "reinitialize" to "reInitialize".

Reply:
redundant with issue below